webapi/src/main/scala/org/knora/webapi/messages/admin/responder/projectsmessages/ProjectsPayloadsADM.scala

```diff
@@ -6,16 +6,14 @@
 package org.knora.webapi.messages.admin.responder.projectsmessages
 
 import zio.json._
-import zio.prelude.Validation
 
-import dsp.errors.ValidationException
 import dsp.valueobjects.Iri.ProjectIri
 import dsp.valueobjects.Project._
 
 /**
  * Project creation payload
  */
-final case class ProjectCreatePayloadADM(
+final case class ProjectCreateRequest(
```

Comment:
Use `ProjectCreatePayload`

Reply:
So how the discussion ended up? This should be done in the separate PR, to make it equal across the API (`payload` is used only in admin routes, `request` vX routes), but here it will get back to `ProjectCreatePayload` w/o ADM suffix?

Reply:
I leave it `Request because in the v2 part of the api and also in the admin section most of our classes representing the body of a request are named `*RequestV2` or `*RequestADM`. The `Payload` postfix is just used a very small portion of second tier model for incoming request which are refining the `String` types from the request to domain types. Since with tapir this step is done by the tapir library we do not need them anymore.